src/hundred_pushups/core.cljc

```diff
@@ -180,10 +180,29 @@
   (time/today-at (time/hour (time.format/parse day-of-formatter time-string)) 00))
 
 (defn day-symbol [day-num]
-  (keyword  (["monday" "tuesday" "wednesday" "thursday" "friday" "saturday" "sunday"] day-num)))
+  (keyword  (["monday" "tuesday" "wednesday" "thursday" "friday" "saturday" "sunday"] (- day-num 1))))
 
 (defn todays-range [whitelist blacklist]
   (map
     parse-time
     (get-in whitelist [(day-symbol (time/day-of-week (time/today)))]))
   )
+
+(defn next-workout-time [options]
+  (let [now-time (get options :current-time time/today)]
```

Comment:
Should this maybe be `(time/today)`? I think as written this will just return the function instead of the value from calling the function.